### PR TITLE
[ios][secure-store] Migrate to expo modules

### PR DESCRIPTION
**Pull request by alanjhughes**
# Why
Continue migrating old modules to Expo modules

# How
Followed typical migration steps

# Test Plan
Tested on real device and simulator. 
NCL ✅

### CODE REVIEW

**Review by fobos531 — CHANGES_REQUESTED** [review]

**Review by Kudo — APPROVED** [review]
just one comment here. otherwise looks good to me. thanks for having the awesome help as always.

File: packages/expo-secure-store/ios/SecureStoreAccessible.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import ExpoModulesCore
+
+enum SecureStoreAccessible: Int, Enumerable {
+  case afterFirstUnlock
+  case afterFirstUnlockThisDeviceOnly
+  case always
+  case whenPasscodeSetThisDeviceOnly
+  case alwaysThisDeviceOnly
+  case whenUnlocked
+  case whenUnlockedThisDeviceOnly
```
[the remark this answers is not in the document]
Comment (alanjhughes) [reply packages/expo-secure-store/ios/SecureStoreAccessible.swift:10]: Thanks, @Kudo 👍 I'll wait for @tsapeta to respond and make the changes then

[the remark this answers is not in the document]
Comment (tsapeta) [reply packages/expo-secure-store/ios/SecureStoreAccessible.swift:10]: Yeah, I agree that explicitly assigning number would make this much more clear.

**Review by Kudo — APPROVED** [review]
🔥 thanks again for helping this migration 🚀

**Pull request merged**